### PR TITLE
Monet

### PR DESCRIPTION
**Pull request by bbakernoaa**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by bbakernoaa** [comment]
Excited to get his going!

File: recipes/monet/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
+{% set name = "monet" %}
+{% set version = "2.1.5" %}
```
Comment (synapticarbors) [inline recipes/monet/meta.yaml]: Mismatch between the version here and the tarball under source

File: recipes/monet/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+package:
+  name: monet
+  version: {{ version }}
+
+source:
+  #git_url: https://github.com/noaa-oar-arl/MONET.git
```
Comment (synapticarbors) [inline recipes/monet/meta.yaml]: delete comments

File: recipes/monet/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  #git_tag: develop
+  url: https://github.com/noaa-oar-arl/MONET/archive/v2.2.0.tar.gz
```
Comment (synapticarbors) [inline recipes/monet/meta.yaml]: Use the jinja templating variable for version in the url so it's easier for the bots to generate a PR when there is an update.

File: recipes/monet/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+  # sha256: 7027a0478cab0f877ebc7e412a0249d0578c0944fab4da6f341175995b957931
+
+build:
+  number: 0
+  #noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
```
Comment (synapticarbors) [inline recipes/monet/meta.yaml]: I don't see any C-extensions being compiled in the `setup.py`, so is the compiler needed?

File: recipes/monet/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+    - python >=3.6
```
Comment (synapticarbors) [inline recipes/monet/meta.yaml]: You don't need `python` listed under `build`. Is this package python3-only though?

File: recipes/monet/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+    - seaborn
+    - pyresample
+    - python-stratify
+    - pydecorate
+    - xesmf  # [not win]
+#    - global_land_mask # [not win]
+    
+test:
+  imports:
+    - monet
```
Comment (synapticarbors) [inline recipes/monet/meta.yaml]: Add imports:

```
- monet.plot
- monet.util
```

File: recipes/monet/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+  home: https://github.com/noaa-oar-arl/monet
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: The Model and ObservatioN Evaluation Tool - for meteorology and AQ
+  doc_url: https://monet-arl.readthedocs.io
+
+extra:
+  recipe-maintainers:
+    # GitHub IDs for maintainers of the recipe.
```
Comment (synapticarbors) [inline recipes/monet/meta.yaml]: delete comments.

File: recipes/monet/meta.yaml [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - python >=3.6
+  host:
+    - python
+    - pip
+  run:
+    - git
```
Comment (synapticarbors) [inline recipes/monet/meta.yaml]: Is `git` needed?

Comment (bbakernoaa) [reply recipes/monet/meta.yaml]: removed

File: recipes/monet/meta.yaml (hunk 7 above)
Comment (bbakernoaa) [reply recipes/monet/meta.yaml]: deleted

File: recipes/monet/meta.yaml (hunk 5 above)
Comment (bbakernoaa) [reply recipes/monet/meta.yaml]: yes it is python 3 only

File: recipes/monet/meta.yaml (hunk 7 above)
Comment (synapticarbors) [reply recipes/monet/meta.yaml]: It looks like the comments are still there under the `recipe-maintainers` section. I think that's the last item and then I can merge this.

File: recipes/monet/meta.yaml (hunk 4 above)
Comment (bbakernoaa) [reply recipes/monet/meta.yaml]: removed

File: recipes/monet/meta.yaml (hunk 3 above)
Comment (bbakernoaa) [reply recipes/monet/meta.yaml]: they have been synced

**Comment by bbakernoaa** [comment]
just removed them and did a push

**Pull request merged**